tests/integration/test_config.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import pytest
+
+from invenio_search import current_search_client as es
+
+
+def _get_suggester_result(response, suggester):
+    return response.json[suggester][0]['options'][0]['_source']
+
+
+def test_literature_suggesters_book_title(api_client, create_record):
+    lit = create_record("lit", data={"titles":[{"title": "Suggested title"}]})
+    es.indices.refresh('records-hep')
```

Comment:
do you need to do this? 🤔 it's already on ``create_record``

Reply:
Actually nope :+1: